docs/plans/2025-12-12-container-version-tracking-implementation.md

```
@@ -0,0 +1,1547 @@
+    digest = models.CharField(
+        max_length=100,
+        unique=True,
+        help_text="SHA256 digest (e.g., sha256:abc123...)",
+    )
+
+    # When we first saw this digest used in a check
+    first_seen_at = models.DateTimeField(auto_now_add=True)
+
+    # Metadata fetched from GHCR/GitHub
+    image_created_at = models.DateTimeField(
+        null=True,
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)